lib/index.js

```diff
@@ -245,28 +240,20 @@ function pollServer(options, callback) {
       if(results.poll.length === 0) {
         return callback();
       }
-      // make the content.holder the new recipient
       // FIXME: do some jsonld processing on message
-      // FIXME: possibly use template configured by top level module to
-      // describe message subject and body
-      // FIXME: perform some validation on the link for safety?
-      // FIXME: the message content should come from a file or template
-      var messages = results.poll.map(function(message) {
-        var newMessage = message;
-        if(message.type === 'CredentialNotification') {
-          // subject and body are ignored if they exist
-          newMessage.recipient = newMessage.content.holder;
-          newMessage.subject = 'A credential is available for pick-up';
-          if(message.content.potentialAction !== null &&
-            message.content.potentialAction.length > 0) {
-            newMessage.content.body = 'Click <a href="' +
-              message.content.potentialAction[0].target.urlTemplate +
-              '" target="_blank">this link</a> to accept the credential.';
-          }
-        }
-        return newMessage;
+      async.auto({
+        emit: callback => {
+          async.each(results.poll, (message, callback) => {
+            bedrock.events.emit(
+              'bedrock-messages-client.' + message.type, message, callback);
```

Comment:
At a minimum we're going to want to do `'bedrock-messages.client.message.' + message.type`.